routers/routes/routes.go

```diff
@@ -489,6 +490,38 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 
+			//Attachement without issue or release attached should not be returned
+			if attach.IsNotAttached() {
```

Comment:
Perhaps this could be improved by writing it like:
```
if attach.IssueID != 0 {
    // ...
} else if attach.ReleaseID != 0 {
    // ...
} else {
    ctx.Error(404)
}
```